.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,175 @@
+---
+name: cloud-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
```

Comment:
I think this is probably a better set of paths -- thoughts @tmdzk @quentinDERORY @andreilee ?

- `MODULE/cloud/**` for each module in https://magma.github.io/magma/docs/next/orc8r/dev_dependencies
- `orc8r/lib/**`

Reply:
Can you already update them for master like @hcgatewood suggested.